schematic/manifest/commands.py

```diff
@@ -168,7 +169,7 @@ def create_single_manifest(data_type, output_csv=None, output_xlsx=None):
             output_path = None
 
         result = manifest_generator.get_manifest(
-            dataset_id=dataset_id, sheet_url=sheet_url, json_schema=json_schema, output_format = output_format, output_path = output_path
+            dataset_id=dataset_id, sheet_url=sheet_url, json_schema=None, output_format = output_format, output_path = output_path
```

Comment:
I thought there was discussion about this but I can't find where. 

See[here](https://github.com/Sage-Bionetworks/schematic/pull/1219/files#diff-6cad414bbd1992ca1cf5ece936af1a8fb6874496d306d3598397247f9b4b9457L128). Currently the code tries to get a value out of the configuration that's never there ("validation schema"), so the value from the configuration will be None always.

But you are correct it can come from the function argument, so I will change it back.